doc/cmp.txt

```diff
@@ -527,6 +527,16 @@ window.{completion,documentation}.zindex~
   `number`
   The completion window's zindex.
 
+                                       *cmp-config.window.completion.col_offset*
+window.completion.col_offset~
+  `number`
+  Offsets the completion window relative to the cursor.
```

Comment:
What if cursor is on column lesser than `col_offset`? Perhaps it should take a function and one co.uld calculate offset taking cursor position (could make function receive it as an argument to make it easier) into consideration

Reply:
With a `col_offset` of `-10`, and the cursor at the first column of a buffer, it won't go over the edge of the window:
![Screenshot 2022-04-14 at 10 33 00](https://user-images.githubusercontent.com/7228095/163346694-b0e4326d-1185-4bb1-ad6d-9f1451a3a3eb.png)

However, if there's a split, it'll be offset by the amount specified:
![Screenshot 2022-04-14 at 10 35 34](https://user-images.githubusercontent.com/7228095/163347009-956761f1-095f-4757-b3f1-571d8b837fba.png)


IMO this is what I would expect.

Reply:
Ah, ok then. Assumed window could go behind visible viewport. Thanks for clarifying. I am wondering whether there is a use case in which function (you could always accept either integer or function) would still be useful.

Reply:
I suppose it's possible, though not for my use case :) I just want to have icons on the left and line the text up nicely